test/puppetlabs/puppetdb/http/events_test.clj

```diff
@@ -29,12 +29,12 @@
 
 (defn get-response
   ([endpoint query]
-     (get-response endpoint query {}))
+   (get-response endpoint query {}))
   ([endpoint query extra-query-params]
-     (let [resp (*app* (get-request endpoint query extra-query-params))]
-       (if (string? (:body resp))
-         resp
-         (update-in resp [:body] slurp)))))
+   (let [resp (*app* (get-request endpoint query extra-query-params))]
+     (if (string? (:body resp))
```

Comment:
Maybe this could just be `(update-in resp [:body] #(if (string? %) % (slurp %)))` since you're digging into the map in the conditional anyways.